### DISCUSSION TITLE
Only 1 Listen address ?

### DISCUSSION THREAD

**Discussion by AdamPerkins** [verdaccio/verdaccio#3969] [Ideas]
When I have multiple Listen addresses in my configuration I see a warning message:  "Warning: multiple listen addresses are deprecated, please use only one".

Why is this deprecated and how can I have in the future an IPv4 and an IPv6 address (dual stack network) to listen on?

**Comment by juanpicado** [comment]
If you provide a good set of examples to open a discussion why this feature should be keep it maybe I reconsider it, but at this point I think does not make any sense to keep it.

**Comment by AdamPerkins** [comment]
As most networks are dual stacked with IPv4 and IPv6 addresses this should be a good reason for. If you use instead of an IP address a domain name with A and AAAA records, a client software has to guess and to try random which IP address is working and which is not. This results in time delay and maybe faults.

Personally I don't know any server software product that is binding only on one single IP address. Supporting IPv4 and IPv6 dual stack is just good practice.

**Reply by juanpicado** [reply]
Thanks for your opinion, I'll wait more feedback at this point will remains as deprecated on 5.x and won't be removed until 6.x (where already was removed (master branch)) but the release is very far away so I'd not worry about it yet, could be put it back easily if that the case.
